src/riak_core_vnode_proxy.erl

```diff
@@ -317,6 +326,10 @@ fake_loop() ->
         {get_count, Pid} ->
             Pid ! {count, erlang:get(count)},
             fake_loop();
+        %% Original tests do not expect replies
```

Comment:
Actually added this for future souls on purpose as I couldn't work out where my replies were.  I'll improve the comment.